packages/qemu-system-x86-64-headless/build.sh

```diff
@@ -78,7 +77,6 @@ termux_step_configure() {
 		--enable-vnc \
 		--disable-vnc-sasl \
 		--enable-vnc-jpeg \
-		--enable-vnc-png \
```

Comment:
Doesn't work? You're still pulling in libpng as a dependency for both packages.

Reply:
Turns out to be just a matter of renaming (to `--enable-png`). Thanks!